.github/workflows/latest-kernel.yml

```diff
@@ -0,0 +1,43 @@
+name: Test build latest kernel available in archlinux
+
+on:
+  workflow_dispatch:
+    inputs:
+      linux-version:
+        description: 'Archlinux kernel version to build a driver against, eg: 6.2.arch1-1'
+        required: false
+        type: string
+  schedule:
+    - cron: '0 8 * * *' # every day at 8am
+
+jobs:
+  build-latest-kernel:
+    name: build-latest-kernel
+    runs-on: ubuntu-latest
+    container:
+      image: falcosecurity/driverkit:latest
+    steps:
+      - name: Checkout Archlinux packages ⤵️
+        uses: actions/checkout@v3
+        with:
+          repository: 'archlinux/svntogit-packages'
+          ref: 'packages/linux'
+          path: 'linux'
+    
+      - name: Fetch latest kernel available on archlinux
+        id: latest-version
+        run: |
+          if [ "${{ inputs.print_tags }}" = "" ]; then
```

Comment:
I don't see any `print_tags` in the inputs section, am I missing something?

Reply:
Ops wrong name!